src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,123 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    private static final String NAME_OF_DRIVER = "name";
+    private static final String DRIVER_LICENSE_NUMBER = "license_number";
+    private static final String ID_OF_DRIVER = "id";
+
+    @Override
+    public Driver create(Driver driver) {
+        String insertDriverQuery =
+                "INSERT INTO drivers(name, license_number) VALUES(?, ?);";
```

Comment:
```suggestion
                "INSERT INTO drivers (name, license_number) VALUES (?, ?);";
```

Please, fix it in all queries